### PR TITLE
added jenn's submission

### PR DESCRIPTION
**Pull request by jennjwang**
I used the best batch size and learning rate from the hyperparameter sweep we did. I increased the number of layers and heads to be 16 and 32. I also used torch.compile.

### CODE REVIEW

**Comment by marcelroed** [comment]
Seems like a bit of an aggressive reformat on the markdown. Can you make your diff just the relevant line? Thanks!

**Comment by jennjwang** [comment]
fixed
